native/cocos/audio/android/utils/Compat.h

```diff
@@ -85,4 +93,23 @@ static inline ssize_t pwrite64(int fd, const void *buf, size_t nbytes, off64_t o
     #define OS_PATH_SEPARATOR '/'
 #endif
 
+#if CC_PLATFORM == CC_PLATFORM_OPENHARMONY
+#ifdef	__cplusplus
+# define __BEGIN_DECLS	extern "C" {
+# define __END_DECLS	}
+#else
+# define __BEGIN_DECLS
+# define __END_DECLS
+#endif
+#endif
+
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
+typedef SLAndroidSimpleBufferQueueItf BufferQueueItf;
+#define IDD_BUFFER_QUEUE SL_IID_ANDROIDSIMPLEBUFFERQUEUE
+#elif CC_PLATFORM == CC_PLATFORM_OPENHARMONY
+typedef SLOHBufferQueueItf BufferQueueItf;
+#define IDD_BUFFER_QUEUE SL_IID_OH_BUFFERQUEUE
```

Comment:
Okay, I found it's a typedef here.

Reply:
Maybe we can use IDD_PLATFORM_BUFFERQUEUE？

Reply:
I think adding `CC_`prefix will make it clear that it's a macro defined in our engine.
```
CC_SL_IDD_BUFFER_QUEUE
```

Reply:
Okay